Bottom: 9ff6ed4c01d9ed3a8f5b5d1e27567d124805dba3
Top:    fda32a70a3dcd3faf576b05f2200c182ce885e7a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 14:25:45 -0500

Refresh of fixup-add-buildbot-ntel-worker

---
diff --git a/ansible-playbook/buildbot.yml b/ansible-playbook/buildbot.yml
index 38127bba..b68da51b 100644
--- a/ansible-playbook/buildbot.yml
+++ b/ansible-playbook/buildbot.yml
@@ -45,6 +45,7 @@
     asset_host: "{{hostvars['nexus']['uri']}}"
     asset_user: "buildbot"
     asset_password: "{{nexus_buildbot_password}}"
+    buildbot_master: "{{ hostvars[buildbot_hostname].ip_addr }}"
     container_etc_hosts:
       nexus.novatech-llc.com: "{{hostvars['nexus']['ip_addr']}}"
     known_hosts: "{{ gitlab_keys.stdout }}"
